routes/users.js

```diff
@@ -122,20 +122,18 @@ router.post('/register', function(req, res, next) {
             db.collection(userDB).insert({ 
                     "username": username,
                     "hash": password,
-                    "email": email
+                    "email": email,
+                    "SEARCH": username.toUpperCase()
                 }, function(err){
                     if(err){
                         res.send("Error processing request");
                     }else{
-                        return res.render('register', {
-                            message:"Go to login to sign in",
-                            type: "success"
-                        }); 
+                        res.redirect('/login');
```

Comment:
there was no good way to pass the alert onto the login page. Before it required the user navigate to the login page after an "account created" alert, now we just direct them to login